libraries/AP_Common/Bitmask.h

```diff
@@ -75,6 +78,10 @@ class Bitmask {
     bool get(uint16_t bit) const {
         uint16_t word = bit/32;
         uint8_t ofs = bit & 0x1f;
+        if (bit >= numbits) {
```

Comment:
it means these header functions are no longer simple
maybe move the functions to *.cpp? Or do this only in SITL?